modules/react-components/src/components/message/message-info.tsx

```diff
@@ -68,14 +73,18 @@ export const MessageInfo: FunctionComponent<MessageInfoProps> = (props: MessageI
     return (
             <Message
                 info
-                header={ (
-                    <Header as='h4'>
-                        <Header.Content>
-                            <Icon name='info circle'/>
-                            { (header) }
-                        </Header.Content>
-                    </Header>
-                ) }
+                onDismiss={ handleCloseInfo }
+                header={
+                    header
+                        ? (
+                            <Header as='h4'>
```

Comment:
Use `"` double quotes and check other places as well.
```suggestion
                            <Header as="h4">
```